schema.yaml

```diff
@@ -385,13 +385,14 @@ variables:
     required: true
     title: "OKE Node Shape"
     description: "Compute shape for nodes in the OKE cluster"
-    default: "VM.Standard2.1"
+    default: "VM.Standard.E3.Flex"
```

Comment:
Using moderately powerful shapes for OKE nodes by default.

Reply:
Suggest you use E4 not E3.